systemtest/src/main/java/io/strimzi/systemtest/Environment.java

```diff
@@ -264,6 +264,10 @@ public static boolean isKRaftModeEnabled() {
         return STRIMZI_FEATURE_GATES.contains(Constants.USE_KRAFT_MODE);
     }
 
+    public static boolean isKafkaNodePoolEnabled() {
```

Comment:
Should it be `isKafkaNodePoolsEnabled`? Both the feature gate and your constant use plural.